plenum/server/replica.py

```diff
@@ -441,6 +441,13 @@ def on_view_change_done(self):
         assert self.isMaster
         self.last_prepared_before_view_change = None
 
+    def on_propagate_primary_done(self):
```

Comment:
Is this code for the scenario where primary crashes and restarts and is still primary since view did not change?